releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -52,12 +52,12 @@
 public class TemplateProcessorTest {
```

Comment:
I am not seeing any new tests especially using `false` for the new boolean parameter in `runPostGenerationAndPublication`. Is it possible to add some tests?

Reply:
I am not sure we have any test on publication as we need to mock Twitter API, it doesn't worth to do this, too expensive for now.